docs/network-flow-visibility.md

```diff
@@ -309,6 +324,10 @@ Please note that the default value for `podLabels` is `false`, which
 indicates source and destination Pod labels will not be included in the flow
 records. If you would like to include them, you can modify the value to true.
 
+Please note the the default value for  `apiPort` is `10348`, which is the port
```

Comment:
s/Please note the the default value/Please note that the default value

Reply:
done